config/iceoryx2.toml

```diff
@@ -1,5 +1,6 @@
 [global]
-root_path                                   = '/tmp/iceoryx2/'
+root_path_unix                              = '/tmp/iceoryx2/'
+root_path_windows                           = 'c:\Temp\iceoryx2\'
```

Comment:
Okay, now I understand. Ideally this could be set during compilation. Maybe a good first issue.